ztp/kube-compare-reference/required/cluster-logging/ClusterLogForwarder.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+{{ .spec | toYaml | indent 2 }}
```

Comment:
For this one there is validation which is critical.
- only one output
- the inputRefs can contain "audit" and "infrastructure" but nothing else (specifically not "application")

Reply:
Added the rules